railseventstore.org/source/docs/exceptions.html.md.erb

```diff
@@ -34,13 +34,21 @@ If you want to have an event present in multiple streams, you have to link it wi
 
 ### RubyEventStore::InvalidExpectedVersion    
 
+<% if version_above('0.30.0') %>
+Occurs when invalid `exception_version` is passed in `append`, `link_to_stream`, `publish_event` or `publish_events`.
```

Comment:
Was `publish` rename merged?

Reply:
https://github.com/RailsEventStore/rails_event_store/pull/377 yes - but I don't want to change docs in this PR - keep it focused on append_to_stream - however this needs update.

Reply:
The changes in docs for #377 are here https://github.com/RailsEventStore/rails_event_store/pull/378